File: src/android/it/cimino/OcrRect.java
package it.cimino;

import org.opencv.core.Rect;

public class OcrRect extends Rect
{
	private Rect area;
	private String text;
	private int confidence;
	
	public OcrRect(Rect area, String text, int confidence){
		this.area = area;
		this.text = text;
		this.confidence = confidence;
	}

	public Rect getArea()
	{
		return area;
	}

	public void setArea(Rect area)
	{
		this.area = area;
	}

	public String getText()
	{
		return text;
	}

	public void setText(String text)
	{
		this.text = text;
	}

	public int getConfidence()
	{
		return confidence;
	}

	public void setConfidence(int confidence)
	{
		this.confidence = confidence;
	}
	
	
}

File: src/ios/OcrRect.m
//
//  OcrRect.m
//  Cimino
//
//  Created by Alessandro Randazzo on 01/12/15.
//
//
#import "OcrRect.h"

@implementation OcrRect

-(cv::Rect)getArea{
	return area;
}

-(void)setArea:(cv::Rect)_area{
	area = _area;
}

-(NSString*)getText{
	return text;
}

-(void)setText:(NSString*)_text{
	text = _text;
}

-(NSInteger)getConfidence{
	return confidence;
}

-(void)setConfidence:(NSInteger)_confidence{
	confidence = _confidence;
}



@end
